.github/workflows/rpk-build.yml

```diff
@@ -13,10 +13,12 @@ on:
     branches: [dev]
     paths:
       - 'src/go/rpk/**'
+      - 'src/v/pandaproxy/schema_registry/protobuf'
       - '.github/workflows/rpk-build.yml'
   pull_request:
     paths:
       - 'src/go/rpk/**'
+      - 'src/v/pandaproxy/schema_registry/protobuf'
```

Comment:
Does this need `/**` at the end?

Reply:
Yes, you are right. Added in https://github.com/redpanda-data/redpanda/compare/f1de4de3714eb845c559fd97cede144b488de8d7..00ea43184a0ad352d0d761f3c87d7b434b4f8695